packages/types/src/block-kit/block-elements.ts

```diff
@@ -161,6 +207,38 @@ export interface FileInput extends Actionable {
   max_files?: number;
 }
 
+/**
+ * @description An icon button to perform actions.
+ */
+export interface IconButton extends Actionable, Confirmable {
+  /**
+   * @description The type of element. In this case `type` is always `file_input`.
+   */
+  type: 'icon_button';
+  /**
+   * @description The icon to show.
+   * @example trash
+   */
+  icon: string;
```

Comment:
question: Today, only `trash` is supported. Do we typically list the supported enum values here?

Reply:
I actually think keeping it open is better as well, because it allows devs to use the SDK while the API evolves with more values!